# Define the goal
Point_xy goal # Specify which dishwasher we want to use
bool flag
---
# Define the result
bool result
---
# Define a feedback message
float64 distance_left